### DISCUSSION TITLE
Some suggestions for planning astrophotography session

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3052] [Ideas]
Hi folks, 

I heavily use Stellarium to plan my imaging sessions beforehand. Over the years the software evolved and I really value its capabilities. To make it even better, I have some suggestions:
1. Add the possibility to add some auxiliary lines in a certain height above the horizon. I only image objects above a certain altitude, and such a line would allow me to quickly check whether all parts of my FOV are above this height without losing the focus on my selected target. This is especially useful for those doing a lot of widefield work, like myself. 
2. Adding a track-like feature that shows where in the sky a selected object will be during a user selectable time span of +/- x hours. This would allow users to quickly judge, whether a selected object will cross certain positions. 

I would really like to see these features in upcoming versions fo Stellarium. 

CS Gerrit

**Comment by gzotti** [comment]
You can use the ArchaeoLines plugin to 

1. show up to two custom altitude circles
2. show the diurnal track (declination track) of the selected object. No time span, but this still helps position estimates. In addition, switch on the "fixed equatorial" (hour angle) grid.

**Comment by GerritErdt** [comment]
Hi, 

many thanks, that did the trick for me!

CS Gerrit
